osmUtils/utils_osm.py

```diff
@@ -268,46 +271,46 @@ def cut_geom(polygon, N):
         
     return intersected_feats
 
-# def get_cut_dfs(multi_pol, infrastructure, filters):
-#     """
-#     Iterates over a cutted geometry and retrieves the cutted parts
-#     Parameters
-#     ----------
-#     multi_pol: 
-#     Returns
-#     --------
-
-#     """
-#     list_dfs = []
-#     for geom in multi_pol:
-#         response_json = download_OSM(geom, infrastructure, filters)
-#         try:
-#             if len(response_json) and response_json['elements']:
-#                 print('Respose_recieved...')
-#                     ## Create graph and convert of GeoDataFrame
-#                 try:
-#                     geoms = OSM_response_to_lines(response_json)
-#                     #df = get_Lines_gdf(G)
-#                     if geoms:
-#                         df = gpd.GeoDataFrame(geometry=geoms)
-#                         list_dfs.append(df)
-#                 except:
-#                     print('Fail generation of graph... No response')
-#         except:
-#             if (response_json==None) or ('remark' in response_json):
-#                 print('response retrieved...')
-#                 multi_pol = cut_geom(geom, 2)
-#                 sublist_dfs = get_cut_dfs(multi_pol,infrastructure, filters)
-#                 list_dfs.append(sublist_dfs)
-#             else:
-#                 print('There is no data for this tile')
-#     #try:         
-#     #    gdf = pd.concat(list_dfs)
-#     #except:
-#     #    print('no objects to concatenate')
-#     #    gdf = None
-#     #
-#     return list_dfs
+def get_cut_dfs(multi_pol, filters):
+    """
+    Iterates over a cutted geometry and retrieves the cutted parts
+    Parameters
+    ----------
+    multi_pol: 
```

Comment:
Should this be a shapely MultiPolygon or a geojson..?

Reply:
Its actually a list of Polygons from the `cut_geom` function. Just make this explicit :)

Reply:
I've updated the docstring a little and renamed `multi_pol` to `polygon_list`